CONTRACT.md

```diff
@@ -0,0 +1,98 @@
+# `cumulus-sled` Contract
+
+`cumulus-sled` is a command-line interface for preparing and outputting Cumulus Messages for Cumulus Tasks. This contract defines how libraries should call `cumulus-sled` to integrate a task into a Cumulus Workflow.
+
+Every Cumulus Task includes a business function. `cumulus-sled` and a language-specific library for interacting with `cumulus-sled` are required to integrate a business function as a Cumulus Task into a Cumulus Workflow.
+
+**NOTE: This should be updated to call `cumulus-sled.zip` once we have created that package.**
```

Comment:
Based on our most recent conversation, this should be removed.  The scripts will just call the cumulus-sled command.